src/__tests__/index.js

```diff
@@ -160,3 +161,15 @@ test(
     shouldThrow,
     `header{background:resemble-image(url("${image}"), twenty-five)}`
 );
+
+test(
+    'should error on non-existing image',
+    shouldThrow,
+    `header{background:resemble-image(url(), twenty-five)}`
+);
+
+test(
+    'should error on unprocessable image',
+    shouldThrow,
+    `header{background:resemble-image(url("${unprocessable}"), twenty-five)}`
```

Comment:
I think I would prefer that we tested this with an empty value for the fidelity, as the module also throws when the fidelity is incorrect. That way we can be sure that we are testing the invalid file and not an invalid fidelity.